doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc

```diff
@@ -22,7 +22,18 @@ endif::[]
 ** Sybase: `org.hibernate.dialect.SybaseASE157Dialect`
 
 .Procedure
-. Unzip the `rhpam-7.4.0-maven-repository.zip` offline Maven repository file.
+. Prepare your database:
+.. Unzip `{PRODUCT_INIT}-{PRODUCT_VERSION}.x-add-ons.zip` in a temporary directory, for example `TEMP_DIR`.
+.. Unzip `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool.zip`.
+.. Change directory to the `TEMP_DIR/{PRODUCT_INIT}-{PRODUCT_VERSION}-migration-tool/ddl-scripts` directory. This directory contains DDL scripts for several database types.
```

Comment:
What are you changing? Are you moving files to the new directory or are you navigating to the other directory to do something?